examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p>A short explanation on how each of the defined schedulers work.</p><p> </p><ul><li><strong>'None'</strong> — apply no decay;</li><li><strong>'exponential_high_frequency' </strong>— apply exponential decay on every epoch with given multiplier;</li><li><strong>'exponential_low_frequency' </strong>—&nbsp;apply exponential decay on each 38th epoch with given multiplier;</li><li><strong>'warmup_two_stage' </strong>—&nbsp;firstly increase learning rate every 6th iteration (apply exponential decay with given multiplier until 900th iteration), than decrease it every 2nd iteration until the end (with new multiplier, also via exponential decay);</li><li><strong>'warmup_three_stage' </strong>— decrease learning rate once, then increase it every 2nd iteration until the 4500th. Do nothing till 5000th, then decrease every 5th iteration to the last. All decays here are also exponential.</li><li><strong>'cyclic' </strong>—&nbsp;apply cyclic decay with given params on each iteration from the beginning to the end.</li></ul><p> </p><p>~~~</p><p> </p><p>Also consider moving this cell below <code>domain</code> definition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>